api.md

```diff
@@ -1874,6 +1868,50 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_continuous_aggregate_policy() :community_function> [](add_continuous_aggregate_policy)
+
+Create a policy that automatically refreshes a continuous aggregate.
+
+#### Required Arguments [](add_continuous_aggregate_policy-required-arguments)
+
+|Name|Description|
+|---|---|
+| `continuous_aggregate` | (REGCLASS) The continuous aggregate to add the policy for. |
+| `start_interval` | Start of the refresh window as an interval relative to the time when the policy is executed. |
+| `end_interval` | End of the refresh window as an interval relative to the time when the policy is executed. |
+| `schedule_interval` | Interval between refresh executions in wall-clock time. |
```

Comment:
Question:  was there discussion to make `schedule_interval` an optional argument, default to to the time_bucket of the continuous agg?

Reply:
There has been discussions about if any of these parameters should be made optional, but for the time being, they are mandatory. If it changes, we have to change the documentation.